### PR TITLE
Reproduction logs + minor suggestions

### PR DESCRIPTION
**Pull request by pjyi2147**
Run on Windows 11 WSL 2 with Ubuntu 22.04. 
Hardware: 5600x 32GB ram with 3060ti 8GB of VRAM.

### CODE REVIEW

File: docs/experiments-msmarco-passage.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ First, we need to download and extract the MS MARCO passage dataset:
 ```bash
 mkdir collections/msmarco-passage
 
-wget https://msmarco.blob.core.windows.net/msmarcoranking/collectionandqueries.tar.gz -P collections/msmarco-passage
+wget https://msmarco.z22.web.core.windows.net/msmarcoranking/collectionandqueries.tar.gz -P collections/msmarco-passage
```
Comment (pjyi2147) [inline docs/experiments-msmarco-passage.md:42]: Update to the link in Anserini. Current link gives me HTTP 409 error.

File: docs/experiments-nfcorpus.md [hunk 2]
```diff
@@ -133,7 +134,7 @@ python -m pyserini.search.faiss \
   --index indexes/nfcorpus.bge-base-en-v1.5 \
   --topics collections/nfcorpus/queries.tsv \
   --output runs/run.beir.bge-base-en-v1.5.nfcorpus.txt \
-  --batch 128 --threads 32 \
+  --batch 128 --threads 8 \
```
Comment (pjyi2147) [inline docs/experiments-nfcorpus.md:137]: I think that thread 32 is too big for personal computers. Performance improves from ~10it/s to ~40it/s on my machine.

File: docs/experiments-nfcorpus.md [hunk 3]
```diff
@@ -91,6 +90,7 @@ python -m pyserini.encode \
 ```
 
 We're using the [`BAAI/bge-base-en-v1.5`](https://huggingface.co/BAAI/bge-base-en-v1.5) encoder, which can be found on HuggingFace.
+Use `--device cuda` for a faster computation if you have a CUDA-enabled GPU.
```
Comment (pjyi2147) [inline docs/experiments-nfcorpus.md:93]: Add a instruction for those who want to run it on GPU. It works on my 3060ti GPU and performance improvement is >7min on CPU to ~20s in GPU.

File: docs/experiments-nfcorpus.md [hunk 4]
```diff
@@ -3,8 +3,7 @@
 This guide contains instructions for running a BGE-base baseline for NFCorpus.
 
 If you're a Waterloo student traversing the [onboarding path](https://github.com/lintool/guide/blob/master/ura.md) (which [starts here](https://github.com/castorini/anserini/blob/master/docs/start-here.md)),
-make sure you've first done the previous step, [a conceptual framework for retrieval
-](conceptual-framework).
+make sure you've first done the previous step, [a conceptual framework for retrieval](https://github.com/castorini/pyserini/blob/master/docs/conceptual-framework.md).
```
Comment (pjyi2147) [inline docs/experiments-nfcorpus.md:6]: Current link was broken.

**Review by lintool — APPROVED** [review]

**Pull request merged**